projects/open62541/build.sh

```diff
@@ -29,40 +29,14 @@ cd $WORK/open62541
 
 cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DUA_ENABLE_AMALGAMATION=OFF \
       -DBUILD_SHARED_LIBS=OFF -DUA_BUILD_EXAMPLES=OFF -DUA_LOGLEVEL=600 \
-      -DUA_ENABLE_DISCOVERY=ON -DUA_ENABLE_DISCOVERY_MULTICAST=ON \
+      -DUA_BUILD_OSS_FUZZ=ON \
       $SRC/open62541/
 
-# for now just use one processor, otherwise amalgamation may fail
+# This also builds all the fuzz targets and places them in the $OUT directory
+# Only build with one process otherwise amalgamation fails.
 make -j1
 
-# ------------------------------------------------------------
-# Add additional definitions which are normally set with CMake
-# ------------------------------------------------------------
-
-# Definitions
-CXXFLAGS="$CXXFLAGS -DUA_NO_AMALGAMATION"
-# Include dirs
-CXXFLAGS="$CXXFLAGS -I$WORK/open62541/src_generated -I$SRC/open62541/include -I$SRC/open62541/plugins -I$SRC/open62541/deps -I$SRC/open62541/src -I$SRC/open62541/src/server"
-
-# ------------------------------------------------------------
-# Build all the fuzzing targets in tests/fuzz
-# ------------------------------------------------------------
-
-fuzzerFiles=$(find $SRC/open62541/tests/fuzz/ -name "*.cc")
-
-for F in $fuzzerFiles; do
-	fuzzerName=$(basename $F .cc)
```

Comment:
just a FYI in case it gets reintroduced: @Dor1s fixed an issue here in https://github.com/google/oss-fuzz/commit/d3168a2c58fa52442f263bf589924be46cf767b2